tour/adts.effekt.md

```diff
@@ -0,0 +1 @@
+../effekt/examples/features/adts.effekt.md
```

Comment:
Just noticed this: Does this mean that the `adt.effekt.md` with a checkfile (singular) isn't linked here?
If so, what's the purpose of that file in `effekt/examples/features`? 🤔

Reply:
When deleting it, we should probably also rename the folder from `features` to `tour`